### PR TITLE
Removes h263 codec code as it is not used.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: src/org/jitsi/impl/neomedia/codec/FMJPlugInConfiguration.java [hunk 1]
```diff
@@ -177,14 +173,6 @@ public static void registerCustomCodecs(boolean enableFfmpeg)
                 "com.sun.media.codec.video.colorspace.RGBScaler",
                 PlugInManager.CODEC);
 
-        // Remove JMF's H263 codec.
```
Comment (bgrozev) [inline src/org/jitsi/impl/neomedia/codec/FMJPlugInConfiguration.java]: If JMF has a 263 impl we probably still want to disable it right?

Comment (damencho) [reply src/org/jitsi/impl/neomedia/codec/FMJPlugInConfiguration.java]: We don't use JMF but fmj and I was seraching for those classes and could not find those, that's why I deleted this

Comment (bgrozev) [reply src/org/jitsi/impl/neomedia/codec/FMJPlugInConfiguration.java]: D'uh of course...sorry for the noise

File: pom.xml [hunk 2]
```diff
@@ -68,7 +68,7 @@
       <groupId>${project.groupId}</groupId>
       <artifactId>jitsi-lgpl-dependencies</artifactId>
       <optional>true</optional>
-      <version>1.1-20190327.160813-5</version>
+      <version>1.1-0-gc27d819</version>
```
Comment (bgrozev) [inline pom.xml]: I wonder if this is considered an older version. Should we bump to 1.2?

Comment (damencho) [reply pom.xml]: https://github.com/jitsi/jitsi-lgpl-dependencies/pull/14

**Pull request merged**